### PR TITLE
Feat : CU-865c8m0vm : Create dashboard

### PR DESCRIPTION
**Pull request by JoHena**
# Fixes ticket [#865c8m0vm](https://app.clickup.com/t/865c8m0vm)

## Problem Description
Create the Dashboard from the available components

## Checklist 
- [x] Passes eslint linter 
- [ ] Passes backend integration tests 
- [x] Added tests for this change 
- [x] Added documentation for this change 
- [ ] Removed unused packages  

## Changes proposed in this pull request
Create Dashboard on /dashboard URL
Create RoutingNav component
Add Google Fonts Import

## Reviewers
- @raulynzunza 
- @jorgelabs 
- @aviada-andreazayas 
- @aviada-fgp 

## Additional Comments

### CODE REVIEW

**Comment by aviada-fgp** [comment]
Task linked: [CU-865c8m0vm Create Dashboard Component](https://app.clickup.com/t/865c8m0vm)

**Review by raulynzunza — APPROVED** [review]

**Review by raulynzunza — APPROVED** [review]

File: src/app/dashboard/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        width="w-[17.375rem]"
+        radius="rounded-[10px]"
+        color="bg-white"
+        boxshadow="shadow-[0_0px_10px]"
+        shadowcolor="shadow-gray-shadow"
+      />
+      <div className="absolute left-[28rem] pt-3">
+        <RoutingNav
+          links={[
+            { urlName: "Home", href: "/Dashboard" },
```
Comment (aviada-andreazayas) [inline src/app/dashboard/layout.tsx:39]: In the design the urlName is Dashboard and the href should be "/dashboard" all in lower case

File: src/app/dashboard/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        classname="left-[7.875rem] top-36"
+        height="h-[58.875rem]"
+        width="w-[17.375rem]"
+        radius="rounded-[10px]"
+        color="bg-white"
+        boxshadow="shadow-[0_0px_10px]"
+        shadowcolor="shadow-gray-shadow"
+      />
+      <div className="absolute left-[28rem] pt-3">
+        <RoutingNav
```
Comment (aviada-andreazayas) [inline src/app/dashboard/layout.tsx:37]: The font color of the routing nav should be secondary (currently is black)

File: src/app/components/Molecules/Hero/Hero.tsx [hunk 3]
```diff
@@ -17,9 +17,11 @@ const Hero: React.FC<IHero> = ({ name, height, url, rounded, padding }) => {
         height: `${height}px`,
         padding: `${padding}px`,
       }}
-      className={`w-full flex items-end`}
+      className={`relative w-full flex items-end`}
     >
-      <h1 className={"text-light font-serif"}>Welcome, {name}</h1>
+      <h1 className={"text-[1.5rem] absolute left-[28rem] text-light font-serif"}>
```
Comment (aviada-andreazayas) [inline src/app/components/Molecules/Hero/Hero.tsx:22]: The font-weight should be bold

**Review by aviada-andreazayas — CHANGES_REQUESTED** [review]

File: src/app/dashboard/layout.tsx (hunk 1 above)
Comment (JoHena) [reply src/app/dashboard/layout.tsx:39]: Fixed all hrefs

File: src/app/components/Molecules/Hero/Hero.tsx (hunk 3 above)
Comment (JoHena) [reply src/app/components/Molecules/Hero/Hero.tsx:22]: Changed font to bold

File: src/app/dashboard/layout.tsx (hunk 2 above)
Comment (JoHena) [reply src/app/dashboard/layout.tsx:37]: Changed font color to secondary

**Review by aviada-andreazayas — APPROVED** [review]

**Pull request merged**